pyproject.toml

```diff
@@ -55,7 +55,7 @@ rich = {version = "^13.2.0", optional = true}
 requests = {version = "^2.28.2", optional = true}
 isort = {version = ">=5.11.4",optional = true}
 black = {version = "22.10.0",optional = true}
-pylink-square = {version = ">=0.12,<0.15", optional = true}
+pylink-square = {version = "~1.2.0", optional = true}
```

Comment:
If you upgrade this then you'll need to add the additional erase_memory step before flashing in jlink_flasher.py